core/src/main/java/io/questdb/std/Unsafe.java

```diff
@@ -248,7 +250,7 @@ public static long malloc(long size, int memoryTag) {
 
     public static long realloc(long address, long oldSize, long newSize, int memoryTag) {
         try {
-            assert memoryTag >= MemoryTag.NATIVE_DEFAULT;
+            assert memoryTag >= MemoryTag.NATIVE_PATH;
```

Comment:
shall we do something about OOM too?

Reply:
Like this? https://github.com/questdb/questdb/pull/4480/commits/2e67428437438794016e27ac56853dd688e4e5e0